### DISCUSSION TITLE
How to access the fields of a relationship that are inside another?

### DISCUSSION THREAD

**Discussion by AxelJPG** [pocketbase/pocketbase#4826] [Q&A]
My tables are related as follows 👇

<img width="777" alt="Screenshot 2024-04-25 002608" src="https://github.com/pocketbase/pocketbase/assets/106621196/59ca84f0-ee7f-4d39-a5fa-bd430c1f9d43">

I am making the data request from the "detail_invoices" table, how can I access the data from the "store" table using expand

**Comment by ganigeorgiev** [comment]
If `detail_invoices.id_producto` and `products.id_negocio` are `relation` fields, then you can simply use dot-notation to target your specific field:

```js
const result = await pb.collection("detail_invoices").getList(1, 30, {
    expand: "id_producto.id_negocio"
})
```
Note that in order the expand to work your client request must satisfy the View API rule of the related `products` and `stores` collections.

For general information how to work with relations please refer to https://pocketbase.io/docs/working-with-relations/.

**Reply by AxelJPG** [reply]
thank you very much, it worked
you are amazing
